dotCMS/src/main/java/com/dotcms/rendering/velocity/viewtools/VelocityWebUtil.java

```diff
@@ -0,0 +1,151 @@
+package com.dotcms.rendering.velocity.viewtools;
+
+import com.dotmarketing.util.*;
+
+import org.apache.velocity.Template;
+import org.apache.velocity.app.VelocityEngine;
+import org.apache.velocity.context.Context;
+import org.apache.velocity.exception.ParseErrorException;
+import org.apache.velocity.exception.ResourceNotFoundException;
+import org.apache.velocity.tools.view.context.ViewContext;
+import org.apache.velocity.tools.view.tools.ViewTool;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.StringWriter;
+
+/**
+ * @author Jason Tesser
+ * @since 1.6.5
+ *
+ */
+public class VelocityWebUtil implements ViewTool {
+
+    static final String _logVariable = "LOG_VELOCITY_TEMPLATES";
+    private Context ctx;
+    private HttpServletRequest req;
+    boolean debug = false;
+
+    // private HttpServletRequest request;
+    public void init(Object obj) {
+        ViewContext context = (ViewContext) obj;
+        this.req = context.getRequest();
+        this.ctx = context.getVelocityContext();
+        this.debug = Config.getBooleanProperty(_logVariable, false);
+
+        if (!this.debug) {
+            try {
+                if (ctx.get("request") != null) {
+                    if (((HttpServletRequest) ctx.get("request")).getParameter(_logVariable) != null
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Merge this if statement with the enclosing one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1066)